### PR TITLE
Added release notes and documentation updates for Forms 13.2.1 and 14.1.1.

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

Added release notes and documentation updates for Forms 13.2.1 and 14.1.1.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [X] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Forms 13.2.1 and 14.1.1

## Deadline (if relevant)

Can be reviewed and published at any time, but at least before 6th August if possible please.

### CODE REVIEW

File: 13/umbraco-forms/developer/working-with-data.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Guid UniqueId
 Dictionary<Guid, RecordField> RecordFields
 ```
 
-In order to access custom Form fields, these are available in the `RecordFields` property. Furthermore there exists an extension method named `ValueAsString` on `IRecord` in `Umbraco.Forms.Core.Services`, such that you can get the value as string given the alias of the field.
+In order to access custom Form fields, these are available in the `RecordFields` property. Furthermore there exists an extension method named `ValueAsString` on `Record` in `Umbraco.Forms.Core.Services`, such that you can get the value as string given the alias of the field.
 
 This extension method handle multi value fields by comma separating the values. E.g. "A, B, C"
```
Comment (eshanrnh) [inline 13/umbraco-forms/developer/working-with-data.md:83]: Should `@using Umbraco.Forms.Core.Services;` in sample code be changed to `@using Umbraco.Forms.Core.Extensions;` as done in v14 article?

File: 13/umbraco-forms/developer/working-with-data.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Guid UniqueId
 Dictionary<Guid, RecordField> RecordFields
 ```
 
-In order to access custom Form fields, these are available in the `RecordFields` property. Furthermore there exists an extension method named `ValueAsString` on `IRecord` in `Umbraco.Forms.Core.Services`, such that you can get the value as string given the alias of the field.
+In order to access custom Form fields, these are available in the `RecordFields` property. Furthermore there exists an extension method named `ValueAsString` on `Record` in `Umbraco.Forms.Core.Services`, such that you can get the value as string given the alias of the field.
```
Comment (eshanrnh) [inline 13/umbraco-forms/developer/working-with-data.md:80]:
```suggestion
In order to access custom Form fields, these are available in the `RecordFields` property. Furthermore there exists an extension method named `ValueAsString` on `Record` in `Umbraco.Forms.Core.Extensions`, such that you can get the value as string given the alias of the field.
```

**Review by eshanrnh — COMMENTED** [review]
Looks good to me, @AndyButland 🙌 I just have 2 suggestions as per the changes done in v14 article 😊

File: 13/umbraco-forms/developer/working-with-data.md (hunk 1 above)
Comment (AndyButland) [reply 13/umbraco-forms/developer/working-with-data.md:83]: No, not for 13.  This moved namespace in 14 so I deliberately only updated in there.

The change on this page is just `IRecord` --> `Record`, which should really have been done when Forms 13 came out, but it wasn't noticed on this page.

Comment (eshanrnh) [reply 13/umbraco-forms/developer/working-with-data.md:83]: Ah ok, Got it 💪

**Pull request merged**